juntagrico/views_create_subscription.py

```diff
@@ -202,9 +203,11 @@ def get_initial(self):
         return {'comment': getattr(self.cs_session.main_member, 'comment', '')}
 
     def get_context_data(self, **kwargs):
+        args = self.cs_session.to_dict()
```

Comment:
could we not put the areas in here 
https://github.com/juntagrico/juntagrico/blob/f12079c51572f9bfc55f653d2bd8543d0212e555/juntagrico/util/sessions.py#L42
?

Reply:
You could, but right now the areas are only used in the summary and nowhere else. Storing it in the sessions is not really necessary.

But it's up to you. This is a quick change.

Reply:
You could, but right now the areas are only used in the summary and nowhere else. Storing it in the sessions is not really necessary.

But it's up to you. This is a quick change.